pkg/render/logstorage/linseed/linseed_test.go

```diff
@@ -159,7 +160,7 @@ var _ = Describe("Linseed rendering tests", func() {
 		})
 
 		It("should render PodAffinity when ControlPlaneReplicas is greater than 1", func() {
-			var replicas int32 = 2
+			replicas = 2
```

Comment:
This is a bit confusing - we default replicas to 2, and then also set it to 2 here. Do we need this line at all?